TheraLang.Web/Controllers/ParticipationController.cs

```diff
@@ -84,21 +83,16 @@ public async Task<ActionResult<IEnumerable<ParticipantViewModel>>> Get()
         [Route("create")]
         public async Task<IActionResult> CreateParticipant([FromBody] int projectId)
         {
-            var userId = User.Claims.GetUserId();
-            if (userId == null)
-            {
-                return BadRequest();
-            }
-
-            var project = await _projectService.GetByIdAsync(projectId);
+            var project = await _projectService.GetById(projectId);
             if (project == null)
             {
                 return NotFound();
             }
 
-            var user = await _userManager.GetUserById(userId.Value);
+            var authUser = await _authenticateService.GetAuthUserAsync();
```

Comment:
So now we would use IAuthenticateService instead of IUserManagementService? If so, why you doesn't removed IUserManagementService dependency from controllers?

Reply:
We dont use it already. Check last commit